src/main/java/org/javarosa/core/model/TriggerableDag.java

```diff
@@ -343,7 +340,7 @@ public void finalizeTriggerables(FormInstance mainInstance, EvaluationContext ev
             }
 
             // save for aggressive 2014 behavior
```

Comment:
Maybe rework this comment while here since there's no more aggressive 2014.

Reply:
I think it's actually easy to follow these values in `immediateCascades` around, so I went ahead and removed the comment.

Reply:
In the next PR (review of collection types), I add a doc block with the details about what side effect each method has. There, I explain how the immediateCascades set gets while building the DAG.